<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Train</class>
 <widget class="QMainWindow" name="Train">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>447</width>
    <height>316</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="TrainWidgets">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>40</y>
      <width>401</width>
      <height>241</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="currentIndex">
     <number>3</number>
    </property>
    <widget class="QWidget" name="DecisionTreeWidget">
     <attribute name="title">
      <string>Decision Tree</string>
     </attribute>
     <widget class="QFrame" name="DecisionTreeForm">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>10</y>
        <width>321</width>
        <height>181</height>
       </rect>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QWidget" name="formLayoutWidget">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>301</width>
         <height>161</height>
        </rect>
       </property>
       <layout class="QFormLayout" name="formLayout">
        <item row="0" column="0">
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Minimum Sample Split </string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLineEdit" name="min_sample_split"/>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Max depth</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QLineEdit" name="max_depth"/>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>Number of features</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QLineEdit" name="n_features"/>
        </item>
        <item row="3" column="1">
         <widget class="QPushButton" name="Confirm">
          <property name="text">
           <string>Confirm Parameters</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="RandomForestWidget">
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <attribute name="title">
      <string>Random Forest</string>
     </attribute>
     <widget class="QFrame" name="RandomForestForm">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>10</y>
        <width>321</width>
        <height>181</height>
       </rect>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QWidget" name="formLayoutWidget_2">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>301</width>
         <height>161</height>
        </rect>
       </property>
       <layout class="QFormLayout" name="formLayout_2">
        <item row="1" column="0">
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>Minimum Sample Split </string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QLineEdit" name="min_sample_split_2"/>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="label_5">
          <property name="text">
           <string>Max depth</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QLineEdit" name="max_depth_2"/>
        </item>
        <item row="3" column="0">
         <widget class="QLabel" name="label_6">
          <property name="text">
           <string>Number of features</string>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="QLineEdit" name="n_features_2"/>
        </item>
        <item row="4" column="1">
         <widget class="QPushButton" name="Confirm_2">
          <property name="text">
           <string>Confirm Parameters</string>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QLabel" name="label_7">
          <property name="text">
           <string>Number of Trees</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLineEdit" name="n_trees"/>
        </item>
       </layout>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="ClaransWidget">
     <attribute name="title">
      <string>CLARANS</string>
     </attribute>
     <widget class="QFrame" name="CLARANSForm">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>10</y>
        <width>321</width>
        <height>181</height>
       </rect>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QWidget" name="formLayoutWidget_3">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>301</width>
         <height>161</height>
        </rect>
       </property>
       <layout class="QFormLayout" name="formLayout_3">
        <item row="1" column="0">
         <widget class="QLabel" name="label_8">
          <property name="text">
           <string>Max Neighbors</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="label_9">
          <property name="text">
           <string>Max Iterations</string>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QLabel" name="label_10">
          <property name="text">
           <string>Distance Metric</string>
          </property>
         </widget>
        </item>
        <item row="4" column="1">
         <widget class="QPushButton" name="Confirm_3">
          <property name="text">
           <string>Confirm Parameters</string>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QLabel" name="label_11">
          <property name="text">
           <string>Number of Medoids</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QSpinBox" name="num_medoids"/>
        </item>
        <item row="1" column="1">
         <widget class="QSpinBox" name="max_neighbors"/>
        </item>
        <item row="2" column="1">
         <widget class="QSpinBox" name="max_iterations"/>
        </item>
        <item row="3" column="1">
         <widget class="QComboBox" name="distance_metric">
          <item>
           <property name="text">
            <string>euclidean</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>manhattan</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>cosine</string>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="DBScanWidget">
     <attribute name="title">
      <string>DBSCAN</string>
     </attribute>
     <widget class="QFrame" name="DBSCANForm">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>10</y>
        <width>321</width>
        <height>181</height>
       </rect>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QWidget" name="formLayoutWidget_4">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>301</width>
         <height>161</height>
        </rect>
       </property>
       <layout class="QFormLayout" name="formLayout_4">
        <item row="0" column="0">
         <widget class="QLabel" name="label_15">
          <property name="text">
           <string>Maximum distance</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QSpinBox" name="eps"/>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_12">
          <property name="text">
           <string>Min Samples</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QSpinBox" name="min_samples"/>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="label_14">
          <property name="text">
           <string>Distance Metric</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QComboBox" name="distance_metric_2">
          <item>
           <property name="text">
            <string>euclidean</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>manhattan</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>cosine</string>
           </property>
          </item>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="QPushButton" name="Confirm_4">
          <property name="text">
           <string>Confirm Parameters</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
